README.md

```diff
@@ -16,7 +14,7 @@ var request = require('superagent'),
 
 request
     .get('/d2l/api/lp/1.5/users/whoami')
-    .use(auth)
+    .use(auth())
```

Comment:
Should we encourage people to just do `var auth = require('superagent-d2l-session-auth')();` instead? Obviously its less flexible but I imagine that would be what most people would want to do anyway? Correct me if I'm wrong.